test/services/teams/index.test.js

```diff
@@ -110,8 +113,61 @@ describe('Test team basic methods', () => {
 					expect(e.message).to.equal('Only administrator, teacher and students can create teams.');
 				});
 			} finally {
-				T.cleanup();
+				cleanup();
 			}
 		});
 	});
 });
+
+describe('Test team extern add', () => {
+	let teacher;
+	let params;
+	let owner;
+	let expert;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['expert' is assigned a value but never used.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=4826661)